docs/components/concepts/outbound-connectors-job-workers.md

```diff
@@ -0,0 +1,64 @@
+---
+id: outbound-connectors-job-workers
+title: Outbound Connectors vs. Job Workers
+description: Integrating with external systems can be done with a Connector or a job worker.
+---
+
+Integrating with external systems can be done with a Connector or a [job worker](job-workers.md).
+
+You define the domain-specific UI for modeling a Connector through a [Connector template](/components/connectors/connector-templates.md). This Connector template is, in fact, an [element template](/components/modeler/desktop-modeler/element-templates/about-templates.md). Therefore, you can also build a Connector-like system using element templates and job workers.
+
+If they both share the same core functionality, how do they differ, and when should you choose what? Connectors and job workers serve different purposes when it comes to aspects like delivery, reusability, focus, and context.
+
+## Delivery
+
+A Connector is reusable code, written as an `OutboundConnectorFunction` using the [Connector SDK](/components/connectors/connector-sdk.md#runtime-logic).
+It is not a standalone application, you cannot start it and have it work on Camunda Platform 8 jobs.
+Instead, a Connector is delivered as a library and can be used in combination with other Connectors in a [Connector runtime environment](/components/connectors/connector-sdk.md#runtime-environments).
+
+In contrast, a job worker is usually part of a Zeebe Client application that can be directly executed to work on jobs.
+
+## Reusability
+
+A job worker usually runs as or inside a standalone Zeebe Client. Without effort, you cannot simply run this in Camunda Platform 8 SaaS or any other environment.
+As a Self-Managed user, you can run it standalone, but often not directly reuse the logic in your existing Zeebe Client that you might already have.
+You can manually extract the job handler from the given job worker, but you also have to ensure that it is still working as expected afterward.
+
+In contrast, a Connector itself is environment-agnostic. There is a runtime environment for Camunda Platform 8 SaaS that can wrap and call this Connector. As the Connector developer, you don't have to worry about this as the runtime takes care of it if you developed the Connector using the Connector SDK.
```

Comment:
I am wondering whether we should mention SaaS here. The runtime exists both for SaaS and SM. We supply a default one but the user is free to bring their own.

Reply:
😄 we have similar review styles